templates/web/base/report/photo.html

```diff
@@ -5,8 +5,8 @@
   [% FOR photo IN object.photos %]
     [% IF c.cobrand.allow_photo_display(object, photo.idx) %]
       <div class="update-img">
-          <a href="[% photo.url_full %]" rel="fancy">
-              <img alt="Photo of this report" src="[% photo.url %]">
+          <a href="[% photo.url_temp_full %]" rel="fancy">
```

Comment:
We do not want to be outputting hash URLs in the general case, that makes them e.g. much harder/impossible to redact once they're picked up by anything else, and this would apply to every photo on the entire site. As we discussed at https://mysociety.slack.com/archives/C01TK8P1K8T/p1727273888229009?thread_ts=1727273574.154349&cid=C01TK8P1K8T this should only be happening when the alert_to_reporter variable is set and we know it's someone who has come direct from an alert email.